### DISCUSSION TITLE
Laravel config issue

### DISCUSSION THREAD

**Discussion by blorange2** [laravel/framework#32818] [General]
I am using Laravel 5.7 and in this scenario I have a config file in the config folder called `forrest.php`, however when dumping `config()` the `forrest` key exists but the values are from a non existent config file.

Anyone experienced similar issues?

**Comment by Lloople** [comment]
Is your config file well formated? Like

```php
<?php

return [

    'key' => 'value'
];
```

Maybe you forgot the return? I've never had issues with custom configuration files actually 🤔
